tests/integration/upgrades/test_local_sharding_upgrades.py

```diff
@@ -186,7 +187,7 @@ async def test_upgrade_cluster(ops_test: OpsTest, righty_upgrade_charm, add_writ
     assert (
         actual_shard_two_writes == shard_two_total_expected_writes
     ), "missed writes during upgrade procedure."
-    logger.error(f"{actual_shard_one_writes = }, {actual_shard_two_writes = }")
+    logger.error(f"{actual_shard_one_writes=}, {actual_shard_two_writes=}")
```

Comment:
Nit: I liked it better with the spaces around which makes it easier to read

Reply:
@Gu1nness I agree but it causes a linting error 😅